### ISSUE TITLE
[Feature Request] Alternative window restore that preserves previous tab states.

### ISSUE THREAD

**Issue by Mansarde** [GarinZ/link-map#37]
Right now, when you `Double Click` a closed window, it simply restores all tabs.
But in Tabs Outliner there is this undocumented feature:
If you `ALT + Double Click` a window, then it only loads the tabs that were open before the window was closed.

<details><summary>Example</summary>
<p>
Let's say opened 10 tabs while doing some research.<br>
During my session I close tabs 3, 4, 5, 8 and 9. So the only tabs left open are 1, 2, 6, 7 and 10.<br>
Then some time later I close/deactivate the window (or even close the browser).<br>
The next day I want to re-open the window, but I don't want to load *all* tabs again, only the ones that weren't already closed!<br>
So I only want tabs 1, 2, 6, 7 and 10 back, because they were open when I last closed/deactivated the window.<br>
In Tabs Outliner I would only have to ALT+Double Click the window for that.
</p>
</details>

### tl;dr
* `Double Click` on window:
Restores all tabs (regardless of their previous state).
=> Already works.

* `ALT + Double Click` on window:
Restores only tabs that were open before the window was closed/deativated.
=> Please implement this. 🙏

**Comment by GarinZ** [comment]
Hi @Mansarde,
As a TabsOutliner user, I didn't even noticed the `Alt + Double Click` make a difference behavior in window opening. So thanks for giving this suggestion. I will try this in TabsOutliner, and add this feature to Link Map. 
For now, Link Map haven't record in detail about closed stated of tabs, so there will need some time to design.

**Comment by Mansarde** [comment]
Hehe, yeah I was surprised as well when I found that out. Can't remember where I read about it, might've been in the Google Group of Tabs Outliner. But once I tried out that feature it became quite useful, as I could keep a lot of tabs deactivated when re-opening the browser (or window).

Thanks for considering, looking forward to it! 🍻
